.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     env:
       MIX_ENV: test
       LIBTORCH_DIR: libtorch
+      LIBTORCH_ZIP: libtorch.zip
```

Comment:
In theory we don't need this var? :)

Reply:
Yeah, although I just realized the cache isn't working... Perhaps @jonatanklosko can help here?

Reply:
You need to update the cache action `with.path` to the new cache directory path, or did I miss it?

Reply:
Let me try this :)
I tested with the updated path, however, just forgot to keep this change in the "final" commit